wgengine/monitor/monitor_linux.go

```diff
@@ -19,6 +19,13 @@ import (
 	"tailscale.com/types/logger"
 )
 
+// unspecifiedMessage is a minimal message implementation that should not
+// be ignored. In general, OS-specific implementations should use better
+// types and avoid this if they can.
+type unspecifiedMessage struct{}
```

Comment:
This doesn't seem right: `monitor_freebsd.go` also uses `unspecifiedMessage`.

Reply:
You might just want to create `monitor_supported.go` which has a `+build` that's a complement of `unsupported` and put this there.